deploy/canary/Dockerfile

```diff
@@ -1,10 +0,0 @@
-FROM golang:latest
```

Comment:
No - that was my original intention but was a bit tricky to achieve while still running on alpine due to glibc linking issues for runc. I'll take another look & see what I can do.